app/src/main/java/com/duckduckgo/app/settings/SettingsViewModel.kt

```diff
@@ -164,7 +164,7 @@ class SettingsViewModel @Inject constructor(
                     appTrackingProtectionOnboardingShown = vpnStore.didShowOnboarding(),
                     appTrackingProtectionEnabled = vpnFeaturesRegistry.isFeatureRegistered(AppTpVpnFeature.APPTP_VPN),
                     emailAddress = emailManager.getEmailAddress(),
-                    showAutofill = autofillStore.autofillAvailable,
+                    showAutofill = autofillCapabilityChecker.canAccessCredentialManagementScreen(),
```

Comment:
is unclear to me that the suggested alternative is better: the management part of credential management seems like a good inclusion than a more general "can access credential screen"

will leave as is since is suggested as a nit only